test/Querier.test.ts

```diff
@@ -29,15 +29,15 @@ const createStore = (
 ): QuerierStoreType => {
   return { key: { id, result, state, $props, $reason } };
 };
-const storeMock: QuerierStoreType = createStore({ state: QuerierState.Active }, null, null);
+const mockStore: QuerierStoreType = createMockStore({ state: QuerierState.Active }, null, null);
```

Comment:
Haha, I guess we misunderstood each other here ;) I meant to change `createStore` to `createStoreMock`. `storeMock` const name was perfectly fine.